tools/azure-sdk-tools/devtools_testutils/envvariable_loader.py

```diff
@@ -62,15 +62,19 @@ def _set_mgmt_settings_real_values(self):
 
             # If environment variables are not all set, check if user-based authentication is requested
             if not all(x is not None for x in [tenant, client, secret]):
-                use_pwsh = os.environ.get("AZURE_TEST_USE_PWSH_AUTH", "false")
-                use_cli = os.environ.get("AZURE_TEST_USE_CLI_AUTH", "false")
-                user_auth = use_pwsh.lower() == "true" or use_cli.lower() == "true"
+                use_pwsh = os.environ.get("AZURE_TEST_USE_PWSH_AUTH", "false").lower()
```

Comment:
This works. If you don't want to only work with `true`, you could use 

```
from ci_tools.functions import discover_targeted_packages, str_to_bool
```

[definition](https://github.com/Azure/azure-sdk-for-python/blob/ab10dfd1835ba432aaa5b2cd656c0429cb765af5/tools/azure-sdk-tools/ci_tools/functions.py#L137)

Reply:
Good call-out! I think I'll update this in a separate PR, to update documentation and other instances as well.